atlp-rwanda/dynamites-ecomm-be
# Fix Deployment Issues by Updating Configuration Settings

## What does this PR do?
This pull request addresses and resolves the deployment issues encountered in our application. The primary changes involve updating the configuration settings to ensure compatibility with our deployment environment.

## Description of Task to be completed?

- Updated configuration settings in config/db.ts
- Modified environment variables to align with the new deployment requirements
- Fixed incorrect paths in the deployment scripts

## How should this be manually tested?
- Checkout this branch `git checkout fix-deployment-issue`
- In your  .env file  update APP_PORT to PORT to ensure proper configuration
- Install all dependencies `npm install`
- Start the server `npm run dev`

## Any background context you want to provide?
Additionally, everyone needs to update their .env files to change the variable APP_PORT to PORT to ensure proper configuration.

## ScreenShoot
<img width="1280" alt="Screenshot 2024-05-24 at 22 13 32" src="https://github.com/atlp-rwanda/dynamites-ecomm-be/assets/88591087/731111a4-ca4b-4c1a-95d4-681b63b38eff">
